### ISSUE TITLE
Multicategory

### ISSUE THREAD

**Issue by nicksavov** [OSTraining/OSDownloads#148]
[feedback]
Hello. Is it possible by some code tweaks to save a file in multiple categories? While creating a new file at the backend, I want multiple select list instead of drop down list. 
[/feedback]
https://www.alledia.com/forum/osdownloads/56/#274

**Comment by htmgarcia** [comment]
This one is hard to acomplish due the categories are managed by the native category manager.

**Comment by andergmartins** [comment]
Yeah, in that case he should use tags instead... and we definitely should support tags (J3+)

**Comment by billtomczak** [comment]
Take it from me - as someone who once created a component using multiple categories - don't do it!

Users loved it, but it created all sorts of complications. +1 on @andergmartins suggestion to use tags. That's what tags were designed for.

**Comment by htmgarcia** [comment]
+1 for tags
